src/marqo/index.py

```diff
@@ -280,61 +280,15 @@ def add_documents(
             non_tensor_fields = []
         if image_download_headers is None:
             image_download_headers = dict()
-        return self._generic_add_update_docs(
+        return self._add_docs_organiser(
             update_method="replace",
```

Comment:
Do we also want to remove the `update_method` parameter? If so, we should remove it from other functions like `_batch_request` as well.

Reply:
Good catch - removed update_method params